roles/ipaserver/library/ipaserver_prepare.py

```diff
@@ -277,6 +280,8 @@ def main():
     options.netbios_name = ansible_module.params.get('netbios_name')
     # additional
     options.setup_ca = ansible_module.params.get('setup_ca')
+    options.random_serial_numbers = ansible_module.params.get(
```

Comment:
There is no leading underscore in this access to `random_serial_numbers`, but the returned value has the leading underscore. Shouldn't there be an underscore here too?

Reply:
This is explained in the commit message ...